projects/js-packages/components/components/alert/index.tsx

```diff
@@ -21,7 +20,17 @@ export const LEVEL_SUCCESS = 'success';
 
 export const ALERT_LEVELS = [ LEVEL_ERROR, LEVEL_WARNING, LEVEL_INFO, LEVEL_SUCCESS ];
 
-const getIconByLevel = level => {
+type AlertProps = {
+	/** The severity of the alert. */
+	level: typeof LEVEL_ERROR | typeof LEVEL_WARNING | typeof LEVEL_INFO | typeof LEVEL_SUCCESS;
```

Comment:
After you add the above suggestion - `as const`, you simply use this

```suggestion
	level: typeof ALERT_LEVELS[ number ];
```

<img width="1094" alt="Screenshot 2022-05-02 at 5 53 26 PM" src="https://user-images.githubusercontent.com/18226415/166233039-40a4b16e-ffb9-4653-8c05-af5e323b194c.png">